CSP/find_domain.py

```diff
@@ -0,0 +1,56 @@
+# This file contains the function find_domain() which will generate the starting domain for one note of a given line
+# with the harmony given as a parameter for context. Also implemented here is the helper function expand_harmony().
+
+import itertools
+
+
+# This function takes a parameter harmony and returns the pitch content of that chord
+# Parameter harmony is a list of the format [root, quality, inversion]
```

Comment:
Why not only take root and quality?  That would make this function more versatile.  It is a bit weird that we accept a parameter (although as part of a list) that we do not use.